examples/tomo/filtered_backprojection_cone_3d.py

```diff
@@ -0,0 +1,74 @@
+# Copyright 2014-2016 The ODL development group
+#
+# This file is part of ODL.
+#
+# ODL is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# ODL is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with ODL.  If not, see <http://www.gnu.org/licenses/>.
+
+"""
+Example using a filtered backprojection (FBP) in cone-beam 3d using `fbp_op`.
+
+Note that the FBP is only approximate in 3d, but still gives a decent
```

Comment:
Well if the kernel is uniform, i.e. a filter, then we get the wrong answer, no? We would need to have some kind of weighting which would make the algorithm e.g. FDK or something else.